provider/cldr/src/transform/locale_canonicalizer/likely_subtags.rs

```diff
@@ -84,36 +84,38 @@ impl From<&cldr_serde::likely_subtags::Resource> for LikelySubtagsV1 {
     fn from(other: &cldr_serde::likely_subtags::Resource) -> Self {
         use icu_locid::LanguageIdentifier;
 
-        let mut language_script: LiteMap<(TinyStr4, TinyStr4), LanguageIdentifier> = LiteMap::new();
-        let mut language_region: LiteMap<(TinyStr4, TinyStr4), LanguageIdentifier> = LiteMap::new();
-        let mut language: LiteMap<TinyStr4, LanguageIdentifier> = LiteMap::new();
-        let mut script_region: LiteMap<(TinyStr4, TinyStr4), LanguageIdentifier> = LiteMap::new();
-        let mut script: LiteMap<TinyStr4, LanguageIdentifier> = LiteMap::new();
-        let mut region: LiteMap<TinyStr4, LanguageIdentifier> = LiteMap::new();
+        let mut language_script: LiteMap<(TinyStr4, TinyStr4), (TinyStr4, TinyStr4, TinyStr4)> =
+            LiteMap::new();
+        let mut language_region: LiteMap<(TinyStr4, TinyStr4), (TinyStr4, TinyStr4, TinyStr4)> =
+            LiteMap::new();
+        let mut language: LiteMap<TinyStr4, (TinyStr4, TinyStr4, TinyStr4)> = LiteMap::new();
+        let mut script_region: LiteMap<(TinyStr4, TinyStr4), (TinyStr4, TinyStr4, TinyStr4)> =
+            LiteMap::new();
+        let mut script: LiteMap<TinyStr4, (TinyStr4, TinyStr4, TinyStr4)> = LiteMap::new();
+        let mut region: LiteMap<TinyStr4, (TinyStr4, TinyStr4, TinyStr4)> = LiteMap::new();
```

Comment:
`nit`: For readability it may be useful to define types like `type ScriptRegion = (TinyStr4, TinyStr4);` and then use them here.

Reply:
What do you think the three-tinystr tuple should be named?

Reply:
Naming it as `LanguageIdentifierStruct`